src/main/java/edu/cornell/kfs/sys/service/impl/CountryServiceImpl.java

```diff
@@ -0,0 +1,72 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import java.text.MessageFormat;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.kuali.kfs.krad.service.BusinessObjectService;
+import org.kuali.kfs.krad.util.ObjectUtils;
+import org.kuali.kfs.sys.businessobject.Country;
+
+import edu.cornell.kfs.sys.CUKFSPropertyConstants;
+import edu.cornell.kfs.sys.service.CountryService;
+
+/**
+ * CU Generic ISO-FIPS Country modification
+ */
+public class CountryServiceImpl implements CountryService {
+
+    private static final Logger LOG = LogManager.getLogger(CountryServiceImpl.class);
+
+    private static final String NO_COUNTRY_FOUND_FOR_CODE = "No Country found for code : {0}";
+    private static final String COUNTRY_CODE_INDICATOR_MESSAGE = "Country code : {0} has status of {1}";
+ 
+    protected BusinessObjectService businessObjectService;
+    
+    public boolean isCountryActive(String countryCode) {
+        Country countryFound = getByPrimaryId(countryCode);
+
+        if (ObjectUtils.isNotNull(countryFound)) {
+            LOG.info("isCountryActive: " +
```

Comment:
The value the user thinks is in error, is the value being logged, so I don't think it will help much.  A developer will need to review the country tables, and the mapping tables.
I was thinking about avoiding extra log entries from the batch log that might not add help.
Is there something I am not considering?